omarchy-plugin/lib/pomo_link/rest.py

```
@@ -0,0 +1,82 @@
+            else:
+                raw = body
+            data = raw
+            headers["Content-Type"] = "application/json"
+        if method == "POST" and data is None:
+            data = b""
+        req = urllib.request.Request(
+            self.url(path, host=host, port=port),
+            data=data,
+            headers=headers,
+            method=method,
+        )
```

Ruff (0.16.2)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)